src/views/MultiView.vue

```diff
@@ -202,6 +202,10 @@ export default {
         },
     },
     mounted() {
+        // Check if layout is empty
+        if (this.layout.length === 0) {
```

Comment:
Add `&& !this.$route.params.layout`, so it doesn't show the preset selector if they are importing a layout from url

Reply:
Added via this commit: [https://github.com/RiceCakess/Holodex/pull/135/commits/a4a828b2befd48cdde8c6548b45126414f7ebd2b](https://github.com/RiceCakess/Holodex/pull/135/commits/a4a828b2befd48cdde8c6548b45126414f7ebd2b)